nshutijonathan/Auto-Mart
# #166152374 Build out User(Seller) mark an AD as sold (UI)

#### What does this PR do?
Have a User(Seller) marking AD page Working
#### Description of Task to be completed?
Have These UI pages working
* Auto-Mart/html/sellerpanel.html
* Auto-Mart/html/markingcars.html
#### How should this be manually tested?
* After cloning the repo, cd into it then  open index.html in the browser or visit [Auto-Mart](https://nshutijonathan.github.io/Auto-Mart/html/)to see all the specified pages
#### Any background context you want to provide?
* There is Implementation of CSS, Javascript, and Html
#### What are the relevant pivotal tracker stories?
[166152374](https://www.pivotaltracker.com/n/projects/2346053/stories/166152374)
--
